### PR TITLE
Make usage of output_dir consistent with cookiecutter

### PR DESCRIPTION
**Pull request by namurphy**
When I was running pieceofcake, I noticed that it was putting the newly created package into a new subdirectory of `output_dir`.  If I set `output_dir` to `./package_name` in the command line, and then also specified the package name as `package_name`, then the new package would be created in ```
```
./package_name/package_name
```
and the new module would be in 
```
./package_name/package_name/module_name
```
The behaviour I was expecting was that I could do
```
pieceofcake gh:OpenAstronomy/packaging_guide .
```
and then have the new package be put into `./package_name` as specified within the program instead of a subdirectory.
    
This commit makes pieceofcake pass `output_dir` directly to cookiecutter without checking if `output_dir` exists and without creating `output_dir`. This way, cookiecutter handles the directory checks & creation instead of pieceofcake.  Before this, `output_dir` was interpreted differentlyby pieceofcake and cookiecutter (partially because the name `output_dir` is ambiguous).
    
If a directory already exists called `package_name` within `output_dir`, then a `cookiecutter.exceptions.OutputDirExistsException` error is raised by cookiecutter.  An alternative possibility would be to catch this exception, but that would be helpful mostly if the cookiecutter error message isn't clear enough.

(By the way, thank you for creating this package!  It has made using cookiecutter easier to understand.)

### CODE REVIEW

File: pieceofcake/__init__.py [hunk 1]
```diff
@@ -1 +1 @@
-from .version import version as __version__
+#from .version import version as __version__
```
Comment (astrofrog) [inline pieceofcake/__init__.py]: @namurphy - was this change intentional?

**Review by astrofrog — APPROVED** [review]
Looks good, thanks! One small comment below about ``__init__.py``

File: pieceofcake/__init__.py (hunk 1 above)
Comment (namurphy) [reply pieceofcake/__init__.py]: Oops!  Nope; the line gave me a problem when I did a local pip install or something (I forgot exactly what) so I commented out rather than trying to figure out what was going on.

**Comment by astrofrog** [comment]
Thanks!
